apps/system-e2e/src/support/email-account.ts

```diff
@@ -55,16 +56,30 @@ async function registerEmailAddressWithSES(emailAccount: {
         'VerificationStatus'
       ] !== 'Success'
     ) {
-      cypressError(`Email identity still not validated in AWS SES`)
+      throw new Error(`Email identity still not validated in AWS SES`)
     }
   } else {
-    cypressError('Verification message not found.')
+    throw new Error('Verification message not found.')
   }
 }
 
-const makeEmailAccount = async () => {
+export type EmailAccount = {
+  getLastEmail(
+    retries: number,
+  ): Promise<{
+    subject: string | undefined
+    text: string | undefined
+    html: string | false
+  } | null>
+  email: string
+}
+const makeEmailAccount = async (name: string): Promise<EmailAccount> => {
   // Generate a new Ethereal email inbox account
-  const testAccount = await createTestAccount()
+  const storage = `${name}-email.json`
```

Comment:
Don't litter my local environment with non-ignored scattered files. IMO should go into a dotfolder

Reply:
👍🏻